rbac/rbac/settings.py

```diff
@@ -287,11 +287,9 @@
         "ecs": {"class": "logging.StreamHandler", "formatter": "ecs_formatter"},
     },
     "loggers": {
-        "django": {"handlers": LOGGING_HANDLERS, "level": DJANGO_LOGGING_LEVEL},
-        "django.server": {"handlers": DEBUG_LOG_HANDLERS, "level": DJANGO_LOGGING_LEVEL},
-        "django.request": {"handlers": DEBUG_LOG_HANDLERS, "level": DJANGO_LOGGING_LEVEL},
```

Comment:
So long as we're using a separate log handler here, DEBUG_LOG_HANDLERS vs LOGGING_HANDLERS these can probably stay, as the problematic data won't get passed to the ecs formatter.